ramls/inventory.raml

```diff
@@ -84,6 +84,28 @@ resourceTypes:
               application/json:
                 example: !include examples/instance_context_get.json
                 schema: context
+  /instancesCollection:
```

Comment:
As a new endpoint is being introduced, please increment the `inventory` interface minor version.

Reply:
As we agreed to move "/instancesCollection" to a separate interface, I propose to stay with current version v8.3 for now